test/widget_test.dart

```diff
@@ -5,10 +5,202 @@
+        (WidgetTester tester) async {
+      // Find the widget
+      /* await tester.pumpWidget(
+      initWidgetToTest(const TitleHeader()),
+    ); */
+
+      await tester.pumpWidget(
+        MaterialApp(
+          home: const Scaffold(
+            body: TitleHeader(),
+          ),
+          routes: {
+            '/about': (context) => const AboutPage(),
+          },
+        ),
+      );
+
+      // Waiting a second to have translations loaded
+      await tester.pump(const Duration(seconds: 1));
+
+      // Test the presence of the logo
+      var appLogo = find.image(const AssetImage('assets/logo.png'));
+      expect(appLogo, findsOneWidget);
+
+      // Test the presence of the app name
+      var appTitle = find.text(tr("appName"));
+      expect(appTitle, findsOneWidget);
+
+      // Test the presence of the about button
+      var aboutButton = find.widgetWithIcon(TextButton, Icons.info_rounded);
+      expect(aboutButton, findsOneWidget);
+
+      // Test the onPressed navigation of the button
+      await tester.tap(aboutButton);
+      await tester.pumpAndSettle();
+
+      expect(find.byType(AboutPage), findsOneWidget);
+    });
+  });
+
+  // ----------------------------ABOUT PAGE---------------------------------- //
+  group('About Page tests', () {
+    testWidgets('About Page has a PageHeader, some Texts and some Linked Icons',
+        (WidgetTester tester) async {
+      // Find the widget
+      await tester.pumpWidget(
+        const MaterialApp(
+          home: Scaffold(
+            body: AboutPage(),
+          ),
+        ),
+      );
+
+      // Waiting a second to have translations loaded
+      await tester.pump(const Duration(seconds: 1));
+
+      // Test the presence of the PageHeader
+      var pageHeader = find.byType(PageHeader);
+      expect(pageHeader, findsOneWidget);
+
+      // Test the presence of the Texts
+      var aboutSubtitle1 = find.text(tr("aboutSubtitle1"));
+      expect(aboutSubtitle1, findsOneWidget);
+      var aboutSubtitle2 = find.text(tr("aboutSubtitle2"));
+      expect(aboutSubtitle2, findsOneWidget);
+      var aboutPart1 = find.textContaining(
+        "${tr("aboutPart1Span1")}${tr("aboutPart1Span2_b")}${tr("aboutPart1Span3")} ${tr("aboutPart1Span4_sb")} ${tr("aboutPart1Span5_s")} ${tr("aboutPart1Span6_sb")}${tr("aboutPart1Span7")}",
+        findRichText: true,
+      );
+      expect(aboutPart1, findsOneWidget);
+      var aboutPart2 = find.textContaining(
+        "${tr("aboutPart2Span1")} ${tr("aboutPart2Span2_sb")} ${tr("aboutPart2Span3_s")} ${tr("aboutPart2Span4_sb")}${tr("aboutPart2Span5")} ${tr("aboutPart2Span6_sb")}${tr("aboutPart2Span7")}",
+        findRichText: true,
+      );
+      expect(aboutPart2, findsOneWidget);
+      var aboutPart3 = find.textContaining(
+        "${tr("aboutPart3Span1")}${tr("aboutPart3Span2_b")} ${tr("aboutPart3Span3_s")} ${tr("aboutPart3Span4_sb")} ${tr("aboutPart3Span5_s")}",
```

Comment:
avoid calling `tr` method inside the test itslef. Use already translated text for tests

Reply:
But it depends on the location if I do that which is less safe ?

Reply:
by the way I tried to change for translated text and it doesn't found any Text widget anymore

Reply:
I found a solution so it's done